### ISSUE TITLE
VertexAI 0.49 release causing issues for VertexAI provider

### ISSUE THREAD

**Issue by romaintoub** [promptfoo/promptfoo#587]
Hello,

I am using `vertex:gemini-pro` as a provider with the following rubric prompt:
```

# The LLM output will be graded by gemini-pro
defaultTest:
  options:
    provider: vertex:gemini-pro
    config:
      temperature: 0
      maxOutputTokens: 1024
    rubricPrompt:
      - role: system
        content: >-
          You are evaluating the answer...
```

However I've installed the latest promptfoo version (0.49) and I am getting totally new results from the grader, most of them being `llm-rubric produced malformed response`. My two cents is that the config for the llm is no longer working but is taking the default parameters from Vertex which is for example `temperature = 0.9` (see https://cloud.google.com/vertex-ai/generative-ai/docs/model-reference/gemini)

**Comment by typpo** [comment]
0.49.0 included a different way of obtaining the Vertex API auth key, but it didn't change the actual request.  Two questions:
-  If you set `LOG_LEVEL=debug`, for example `LOG_LEVEL=debug promptfoo eval`, what is in the API response?
- Does switching to 0.48.0 (`npx promptfoo@0.48.0 eval`) solve the problem and if so, how does the response differ?

**Comment by typpo** [comment]
Passthrough of generationConfig options should be fixed in 0.49.2
